frontend/public/components/filter-toolbar.tsx

```diff
@@ -294,49 +293,51 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
             )}
           </ToolbarItem>
         )}
-        <ToolbarItem className="co-filter-search--full-width">
-          <ToolbarFilter
-            deleteChipGroup={() => updateLabelFilter([])}
-            chips={[...labelFilters]}
-            deleteChip={(filter, chip: string) =>
-              updateLabelFilter(_.difference(labelFilters, [chip]))
-            }
-            categoryName="Label"
-          >
+        {(!hideToolbar || !hideLabelFilter) && (
+          <ToolbarItem className="co-filter-search--full-width">
             <ToolbarFilter
-              chips={nameFilter && nameFilter.length > 0 ? [nameFilter] : []}
-              deleteChip={() => updateNameFilter('')}
-              categoryName="Name"
+              deleteChipGroup={() => updateLabelFilter([])}
+              chips={[...labelFilters]}
+              deleteChip={(filter, chip: string) =>
+                updateLabelFilter(_.difference(labelFilters, [chip]))
+              }
+              categoryName="Label"
             >
-              <div className="pf-c-input-group">
-                {!hideLabelFilter && (
-                  <DropdownInternal
-                    items={FilterType}
-                    onChange={switchFilter}
-                    selectedKey={filterType}
-                    title={filterType}
-                  />
-                )}
-                {!hideToolbar && (
-                  <AutocompleteInput
-                    className="co-text-node"
-                    onSuggestionSelect={(selected) => {
-                      updateLabelFilter(_.uniq([...labelFilters, selected]));
-                    }}
-                    showSuggestions={FilterType.LABEL === filterType}
-                    textValue={inputText}
-                    setTextValue={updateSearchFilter}
-                    placeholder={
-                      FilterType.NAME === filterType ? 'Search by name...' : 'app=frontend'
-                    }
-                    data={data}
-                    labelPath={props.labelPath}
-                  />
-                )}
-              </div>
+              <ToolbarFilter
+                chips={nameFilter && nameFilter.length > 0 ? [nameFilter] : []}
+                deleteChip={() => updateNameFilter('')}
+                categoryName="Name"
+              >
+                <div className="pf-c-input-group">
+                  {!hideLabelFilter && (
+                    <DropdownInternal
+                      items={FilterType}
+                      onChange={switchFilter}
+                      selectedKey={filterType}
+                      title={filterType}
+                    />
+                  )}
+                  {!hideToolbar && (
```

Comment:
This check is no longer needed with the addition of the new one above.

Reply:
done